### PR TITLE
Bump everything to 1.8.9 Retroarch release head

### PR DESCRIPTION
**Pull request by Ntemis**
- Tinkerboard: Build tested with unbuildable cores disabled

### CODE REVIEW

**Comment by ToKe79** [comment]
Hi,
this PR (dd80207825915f995ed898e70698a4577cd4d272) did not build correctly for any target - all (16) builds failed.
Logs can be found here: http://nightly.builds.lakka.tv/PRs/pr_971_000156/

_This is an automated message_

File: projects/Rockchip/devices/TinkerBoard/options [hunk 1]
```diff
@@ -28,3 +28,9 @@
 
   # kernel serial console
     EXTRA_CMDLINE="console=uart8250,mmio32,0xff690000 console=tty0"
+
```
Comment (RobLoach) [inline projects/Rockchip/devices/TinkerBoard/options:31]: May be good to stick a comment in there letting others know why these are here...
``` suggestion

# Disabled Cores
```

Comment (Ntemis) [reply projects/Rockchip/devices/TinkerBoard/options:31]: It's not permanent and this has to be done for all targets

**Comment by ToKe79** [comment]
Hi,
this PR (dd80207825915f995ed898e70698a4577cd4d272) did not build correctly for any target - all (16) builds failed.
Logs can be found here: http://nightly.builds.lakka.tv/PRs/pr_971_000157/

_This is an automated message_

**Pull request merged**